More sensors

alexLouw opened the pull request:
added changes necessary for more sensors.

alexLouw committed:
Introduced use of 2 more sensors and a dictionary for actions

alexLouw committed:
Added 2 missing if conditions

ChielBruin committed:
Fixed some stuff and added TODO tags

ChielBruin commented:
I fixed some syntax errors

ChielBruin commented on catkin_ws/src/kpr_arm_control/src/py_arm_control/states.py:
```diff
@@ -56,8 +68,9 @@ def execute(self, userdata):
-			print(self.setIO(2, True, 2, False))
+			# TODO missing if here
+			#print(self.setIO(GRIPPER_OUT, GRIPPER_OPEN, GRIPPER_IN_OPENED, True))
+			print(self.setIO(*ACTIONS["OpenGripper"]))
```
Missing if

ChielBruin commented on catkin_ws/src/kpr_arm_control/src/py_arm_control/states.py:
```diff
@@ -109,9 +123,14 @@ def execute(self, userdata):
+			#if self.setIO(GRIPPER_OUT, GRIPPER_OPEN, GRIPPER_IN_OPENED, True): #2,True, 2, False
+			if (self.setIO(*ACTIONS["OpenGripper"])):
+				userdata.result.release = HarvestStatus(success = HarvestStatus.OK, message = 'Successfully opened the gripper')
+				# TODO missing if here
```
Missing if

ChielBruin reviewed (changes requested):
You can also remove the old setIO code you commented out, as this new notation is much more readable and therefore the old code is not needed for reference purposes

ChielBruin committed:
Probably a real fix this time

alexLouw committed:
Processed TODO's

Introduced 2 more if statements, removed the commented IO-control lines
and introduced an release if statement for release order based on the
cucumbers orientation (manually altered in file)

ChielBruin committed:
Angle is 30 degrees again

ChielBruin committed:
Merge branch 'testing' into moreSensors

ChielBruin committed:
Some more improvements to the state machine
